### PR TITLE
{data,math}[foss/2020a] causalml v0.8.0-20200909, Keras v2.3.1 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in total)
node353.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/0a3c91cbcdfc2a091fbbf09ba296df7c for a full test report.

**Comment by smoors** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node375.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/d7a61522b2a73efc9e80aa84afe3a2f4 for a full test report.

**Comment by smoors** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/c/causalml/causalml-0.8.0-20200909-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,74 @@
+easyblock = 'PythonBundle'
+
+name = 'causalml'
```
Comment (lexming) [inline easybuild/easyconfigs/c/causalml/causalml-0.8.0-20200909-foss-2020a-Python-3.8.2.eb:3]: Sorry to be picky about names, but `CasualML` or `Casual-ML` seem more fitting here :slightly_smiling_face:

File: easybuild/easyconfigs/c/causalml/causalml-0.8.0-20200909-foss-2020a-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/uber/causalml/archive/'],
+        'checksums': ['b33401f89e9e7e38c5d590ca349387e3c91c609164d5b391cefed980e39b4387'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/causalml'],
+}
```
Comment (lexming) [inline easybuild/easyconfigs/c/causalml/causalml-0.8.0-20200909-foss-2020a-Python-3.8.2.eb:70]: These sanity checks are default with `PythonBundle`
```suggestion
```

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/causalml/causalml-0.8.0-20200909-foss-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/c/causalml/causalml-0.8.0-20200909-foss-2020a-Python-3.8.2.eb:3]: it was not chosen by me, there's already one other version available. `causalml` is also the name used on pypi.org.
do you think we should change both or leave it as is?

Comment (lexming) [reply easybuild/easyconfigs/c/causalml/causalml-0.8.0-20200909-foss-2020a-Python-3.8.2.eb:3]: ah yeah, I missed the one already in the tree. Then I would not bother with changing the name, `causalml` is good enough.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/b48a80fcf39a5f43d4c955d00b7db2b0 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 13 out of 13 (2 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.6
See https://gist.github.com/db0799d66d7c44971fa521c2f7440e79 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @smoors!

**Pull request merged**